src/footer.test.jsx

```diff
@@ -116,9 +115,28 @@ describe('LanguageSelectInput', () => {
 });
 
 describe('SocialIconsList', () => {
-	it('exists', () => {
-		const socialIconList = shallow(<SocialIconsList language={FRENCH_LANG} />);
-		expect(socialIconList).toMatchSnapshot();
+	const allLocales = [
+		'en-US',
+		'en-AU',
+		'de-DE',
+		'es',
+		'es-ES',
+		'fr-FR',
+		'it-IT',
+		'nl-NL',
+		'pl-PL',
+		'pt-BR',
+		'sv-SE',
+		'tr-TR',
+		'th-TH',
+		'ja-JP',
+		'ko-KR',
+	];
+	allLocales.forEach(localeCode => {
+		it(`renders for ${localeCode} locale`, () => {
```

Comment:
Not sure if this is a good way to do this or not. Open to suggestions.

Reply:
LGTM